src/signers/jwt-signer.ts

```diff
@@ -45,7 +49,7 @@ export class AbstractAccountJWTSigner extends AASigner {
     const hashSignBytes = sha256(signBytes);
     const message = Buffer.from(hashSignBytes).toString("base64");
 
-    const authResponse = await fetch('https://burnt-abstraxion-api.onrender.com/api/v1/sessions/authenticate', {
+    const authResponse = await fetch(`${this.apiUrl}/api/v1/sessions/authenticate`, {
```

Comment:
Let me know if we should keep this format, or pass in the full url (w/ endpoint)